frontend/src/components/Header.vue

```diff
@@ -0,0 +1,21 @@
+<template lang="pug">
+    v-app-bar(app clipped-left color="#000080")
```

Comment:
Define any new color in https://github.com/devlup-labs/portfolio/blob/master/frontend/src/plugins/vuetify.js